src/CSI.IBTA.UserService/Services/EmployeesService.cs

```diff
@@ -42,5 +45,67 @@ public async Task<GenericResponse<PagedEmployeesResponse>> GetEmployees(int page
             var response = new PagedEmployeesResponse(employeeDtos, page, pageSize, totalPages, totalCount);
             return new GenericResponse<PagedEmployeesResponse>(null, response);
         }
+
+        public async Task<GenericResponse<EmployeeDto>> CreateEmployee(CreateEmployeeDto dto)
+        {
+            bool hasSameSSN = await _userUnitOfWork.Users.GetSet().AnyAsync(x => x.SSN == dto.SSN);
+            if (hasSameSSN)
+            {
+                return new GenericResponse<EmployeeDto>(new HttpError("An employee already exists with the same SSN.", HttpStatusCode.BadRequest), null);
+            }
+
+            bool hasSameName = await _userUnitOfWork.Users.GetSet().AnyAsync(x => x.Firstname == dto.FirstName && x.Lastname == dto.LastName);
+            if (hasSameName)
+            {
+                return new GenericResponse<EmployeeDto>(new HttpError("An employee already exists with the same name.", HttpStatusCode.BadRequest), null);
+            }
+
+            bool hasSameUsername = await _userUnitOfWork.Users.GetSet().AnyAsync(x => x.Account.Username == dto.UserName);
+            if (hasSameUsername)
+            {
+                return new GenericResponse<EmployeeDto>(new HttpError("An employee already exists with the same username.", HttpStatusCode.BadRequest), null);
+            }
+
+            var e = new User()
```

Comment:
```suggestion
            var user = new User()
```
lets have semantic namings